dash-cost-dashboard.py

```diff
@@ -126,9 +172,13 @@ def get_execution_cost_table(aggregated_allocations: List) -> DataFrame:
             "COMPUTE COST": compute_cost,
             "MEMORY COST": ram_cost,
             "STORAGE COST": storage_cost,
-            "TOTAL COST": total_cost
+            "ALLOC COST": alloc_total_cost
         })
-    execution_costs = pd.DataFrame(exec_data)
+    execution_costs = distribute_cost(pd.DataFrame(exec_data))
+    cloud_cost_diff = cloud_cost - execution_costs["ALLOC COST"].sum()
+    
+    execution_costs['CLOUD COST'] = execution_costs['ALLOC COST'] / execution_costs["ALLOC COST"].sum() * cloud_cost_diff
+    execution_costs['TOTAL COST'] = execution_costs['ALLOC COST'] + execution_costs['CLOUD COST']
```

Comment:
I had to squint at this pretty hard, consider more explanatory variable names, or maybe a comment or something.  It's not a big deal
```suggestion
    accounted_cost = execution_costs["ALLOC_COST"].sum()
    cloud_cost_unaccounted = cloud_cost - accounted_cost

    execution_costs['CLOUD COST'] = cloud_cost_unaccounted * execution_costs['ALLOC COST'] / accounted_cost
    execution_costs['TOTAL COST'] = execution_costs['ALLOC COST'] + execution_costs['CLOUD COST']
```